invenio_communities/assets/semantic-ui/js/invenio_communities/search/index.js

```diff
@@ -0,0 +1,41 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2020 CERN.
+// Copyright (C) 2020 Northwestern University.
+// Copyright (C) 2021 Graz University of Technology.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+
+import $ from "jquery";
+import { createSearchAppInit } from "@js/invenio_search_ui";
+import {
+  RDMBucketAggregationElement,
+  RDMRecordFacets,
+  RDMRecordFacetsValues,
+  RDMRecordResultsGridItem,
+  RDMRecordResultsListItem,
+  RDMRecordSearchBarContainer,
+  RDMRecordSearchBarElement,
+  RDMCountComponent,
+  RDMEmptyResults,
+  RDMErrorComponent,
+} from "./components";
+
+const initSearchApp = createSearchAppInit({
+  "BucketAggregation.element": RDMBucketAggregationElement,
+  "BucketAggregationValues.element": RDMRecordFacetsValues,
+  "ResultsGrid.item": RDMRecordResultsGridItem,
+  "EmptyResults.element": RDMEmptyResults,
+  "ResultsList.item": RDMRecordResultsListItem,
+  "SearchApp.facets": RDMRecordFacets,
+  "SearchApp.searchbarContainer": RDMRecordSearchBarContainer,
+  "SearchBar.element": RDMRecordSearchBarElement,
+  "Count.element": RDMCountComponent,
+  "Error.element": RDMErrorComponent,
+});
+
+$(".community-description-button").on('click', function () {
```

Comment:
shouldn't this be bound to id, not class of an element?

Reply:
With the current implementation, there are two buttons with different name but same functionality (it makes sense, because both buttons want to toggle the same text, but for the user, the effect will be the opposite). As IDs are unique, using a class selector should be the best way to avoid duplicating the selector logic.

Reply:
what if someone will give this class to any other element? 
You can assign two different ids, to one event in JS

Reply:
what if someone will give this class to any other element? 
You can assign two different ids, to one event in JS